python/ray/tune/suggest/dragonfly.py

```diff
@@ -331,17 +330,6 @@ def on_trial_complete(self,
             self._opt.tell([(trial_info,
                              self._metric_op * result[self._metric])])
 
-    def save(self, checkpoint_path: str):
-        trials_object = (self._initial_points, self._opt)
-        with open(checkpoint_path, "wb") as outputFile:
-            pickle.dump(trials_object, outputFile)
-
-    def restore(self, checkpoint_dir: str):
-        with open(checkpoint_dir, "rb") as inputFile:
-            trials_object = pickle.load(inputFile)
-        self._initial_points = trials_object[0]
-        self._opt = trials_object[1]
-
```

Comment:
Dragonfly has unpickable subfunctions in the multi function runner. We should disable saving/restoring for now.